sda-commons-server-dropwizard/src/test/java/org/sdase/commons/server/dropwizard/bundles/DefaultLoggingConfigurationBundleWithRequestLogFilterTest.java

```diff
@@ -0,0 +1,47 @@
+package org.sdase.commons.server.dropwizard.bundles;
+
+import static io.dropwizard.testing.ResourceHelpers.resourceFilePath;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.dropwizard.Configuration;
+import io.dropwizard.logging.ConsoleAppenderFactory;
+import io.dropwizard.request.logging.LogbackAccessRequestLogFactory;
+import io.dropwizard.request.logging.filter.UriFilterFactory;
+import io.dropwizard.server.DefaultServerFactory;
+import io.dropwizard.testing.junit.DropwizardAppRule;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.sdase.commons.server.dropwizard.bundles.test.LoggingTestApp;
+import org.sdase.commons.server.testing.SystemPropertyRule;
+
+public class DefaultLoggingConfigurationBundleWithRequestLogFilterTest {
```

Comment:
I think you can even write a test that makes sure the expected lines are not logged. You can use JUnit Pioneer to access standard out: https://junit-pioneer.org/docs/standard-input-output/